### PR TITLE
[Az.RecoveryServices.Backup] Workload Recovery config, disallowing target item to be an availability group

### PR DESCRIPTION
**Pull request by hiaga**
## Description
- Workload Recovery config, disallowing target item to be an availability group [bug fix]

## Checklist

- [Yes] I have read the [_Submitting Changes_](../blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/master/CONTRIBUTING.md)
- [Yes] The title of the PR is clear and informative
- [Yes] The appropriate `ChangeLog.md` file(s) has been updated:
    - For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    - A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header -- no new version header should be added
- [N/A] The PR does not introduce [breaking changes](../blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [N/A] If applicable, the changes made in the PR have proper test coverage
- [N/A] For public API changes to cmdlets:
    - [N/A] a cmdlet design review was approved for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (_Microsoft internal only_)
        - {Please put the link here}
    - [N/A] the markdown help files have been regenerated using the commands listed [here](../blob/master/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)

### CODE REVIEW

**Comment by hiaga** [comment]
@msJinLei, @wyunchi-ms , @isra-fel : could someone please review this PR

File: src/RecoveryServices/RecoveryServices.Backup/Cmdlets/Restore/GetAzureRmRecoveryServicesBackupWorkloadRecoveryConfig.cs [hunk 1]
```diff
@@ -127,6 +126,16 @@ public override void ExecuteCmdlet()
                 TimeSpan timeSpan = DateTime.UtcNow - new DateTime(1970, 1, 1);
                 int offset = (int)timeSpan.TotalSeconds;
                 string targetDb = "";
+
+                if (TargetItem != null)
+                {
+                    if (string.Compare(((AzureWorkloadProtectableItem)TargetItem).ProtectableItemType,
```
Comment (isra-fel) [inline src/RecoveryServices/RecoveryServices.Backup/Cmdlets/Restore/GetAzureRmRecoveryServicesBackupWorkloadRecoveryConfig.cs:132]: please use `!string.Equals(...)` BTW

File: src/RecoveryServices/RecoveryServices/ChangeLog.md [hunk 2]
```diff
@@ -18,6 +18,7 @@
         - Additional information about change #1
 -->
 ## Upcoming Release
+* Workload config, blocking target item to be an availability group.
```
Comment (isra-fel) [inline src/RecoveryServices/RecoveryServices/ChangeLog.md:21]: How about
```suggestion
* Blocked getting workload config when target item is an availability group.
```

**Review by isra-fel — CHANGES_REQUESTED** [review]

**Review by isra-fel — APPROVED** [review]
LGTM

**Pull request merged**